ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js

```diff
@@ -18,15 +17,22 @@ export default {
       control: 'array',
       table: { category: 'Data' },
     },
+    symbol: {
+      control: 'text', // specify the control type as 'text'
+      defaultValue: 'META', // set the default value
+      table: { category: 'Data' },
+    },
+    image: {
+      control: 'text', // specify the control type as 'text'
+      defaultValue: 'metamark.svg', // set the default value
+      table: { category: 'Data' },
+    },
   },
```

Comment:
`defaultValue` is deprecated please use default args
```suggestion
    symbol: {
      control: 'text',
      table: { category: 'Data' },
    },
    image: {
      control: 'text',
      table: { category: 'Data' },
    },
  },
  args: {
    symbol: 'ETH',
    image: './images/eth_logo.svg',
  },
```
I also think its probably best to make the default eth